### PR TITLE
raspbian package settings

### PR DESCRIPTION
**Pull request by hpsaturn**
changed dependencies and debian rules for proto namespaces

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -14,33 +14,28 @@ FIND_LIBRARY(ZMQ_LIB NAMES zmq)
...
   driver_manager.cpp driver_manager.h
-  ${MALOS_PROTO_SRC} ${MALOS_PROTO_HEADER}
 )
 
 add_library(matrix_malos ${matrix_malos_src})
-set_property(TARGET matrix_malos PROPERTY CXX_STANDARD 11)
-target_link_libraries(matrix_malos matrix_malos_zmq)
-target_link_libraries(matrix_malos ${PROTOBUF_LIBRARY} ${CMAKE_THREAD_LIBS_INIT})
+  set_property(TARGET matrix_malos PROPERTY CXX_STANDARD 11)
+  target_link_libraries(matrix_malos matrix_malos_zmq)
```
Comment (arhuaco) [inline src/CMakeLists.txt:36]: Why the leading spaces?

**Review by arhuaco — APPROVED** [review]
Cool. Please check a comment I left.

File: src/CMakeLists.txt (hunk 1 above)
Comment (hpsaturn) [reply src/CMakeLists.txt:36]: mmm, I don't know, maybe @eljuguetero knows ?

Comment (eljuguetero) [reply src/CMakeLists.txt:36]: feng shui, please keep the spaces.

Comment (arhuaco) [reply src/CMakeLists.txt:36]: :-P

**Pull request merged**